sdk/test-utils/perfstress/src/testProxyHttpClient.ts

```diff
@@ -0,0 +1,272 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { HttpOperationResponse } from "@azure/core-http";
+import { DefaultHttpClient, WebResourceLike } from "@azure/core-http";
+import {
+  PipelinePolicy,
+  PipelineRequest,
+  PipelineResponse,
+  SendRequest
+} from "@azure/core-rest-pipeline";
+import { Dispatcher, request } from "undici";
```

Comment:
do we really need to drag in an entire new http stack just to ping a couple endpoints? This package is node-only, right? Can't we just use the "http" module if you don't want to use the default clients?

Reply:
Don't get me wrong, I love `undici`, I just am allergic to adding more deps to our rush package graph, especially for test-only packages.

Reply:
Done!